CHANGELOG.md

```diff
@@ -2,6 +2,264 @@
+```
+
+By [@andrewmcgivery](https://github.com/andrewmcgivery) in https://github.com/apollographql/router/pull/7016
+
+### Traffic shaping for connectors ([PR #6737](https://github.com/apollographql/router/pull/6737))
+
+Traffic shaping is now supported for connectors. To target a specific source, use the `subgraph_name.source_name` under the new `connector.sources` property of `traffic_shaping`. Settings under `connector.all` will apply to all connectors. `deduplicate_query` is not supported at this time.
+
+Example config:
+
+```
+traffic_shaping:
+  connector:
+    all:
+      timeout: 5s
+    sources:
+      connector-graph.random_person_api:
+        global_rate_limit:
+          capacity: 20
+          interval: 1s
+        experimental_http2: http2only
+        timeout: 1s
+```
+
+By [@andrewmcgivery](https://github.com/andrewmcgivery) in https://github.com/apollographql/router/pull/6737
+
+### Add apollo.router.pipelines metrics ([PR #6967](https://github.com/apollographql/router/pull/6967))
+
+When the Router reloads either via schema change or config change a new request pipeline is created.
+Existing request pipelines are closed once their requests finish. However, this may not happen if there are ongoing long requests 
+such as Subscriptions that do not finish.
+
+To enable debugging when request pipelines are being kept around a new gauge metric has been added:
+
+- `apollo.router.pipelines` - The number of request pipelines active in the router
+    - `schema.id` - The Apollo Studio schema hash associated with the pipeline.
+    - `launch.id` - The Apollo Studio launch id associated with the pipeline (optional).
+    - `config.hash` - The hash of the configuration
+
+By [@BrynCooke](https://github.com/BrynCooke) in https://github.com/apollographql/router/pull/6967
+
+### Update JWT handling ([PR #6930](https://github.com/apollographql/router/pull/6930))
+
+This PR updates JWT-handling in the `AuthenticationPlugin`;
+
+- Users may now set a new config option `config.authentication.router.jwt.on_error`.
+  - When set to the default `Error`, JWT-related errors will be returned to users (the current behavior). 
+  - When set to `Continue`, JWT errors will instead be ignored, and JWT claims will not be set in the request context.
+- When JWTs are processed, whether processing succeeds or fails, the request context will contain a new variable `apollo::authentication::jwt_status` which notes the result of processing.
+
+By [@Velfi](https://github.com/Velfi) in https://github.com/apollographql/router/pull/6930
+
+### Support to get/set URI scheme in Rhai ([Issue #6897](https://github.com/apollographql/router/issues/6897))
+
+This adds support to read the scheme from the `request.uri.scheme`/`request.subgraph.uri.scheme` functions in Rhai, 
+enabling the ability to switch between http and https for subgraph fetches. For example:
+
+```rs
+fn subgraph_service(service, subgraph){
+    service.map_request(|request|{
+        log_info(``);
```

Comment:
Our changeset script automatically templatized a thing here and swallowed it (Rhai and Changesets both use the same thing!)

```suggestion
        log_info(`${request.subgraph.uri.scheme}`);
```